crates/forge-runner/src/test_crate_summary.rs

```diff
@@ -8,28 +8,31 @@ pub struct TestCrateSummary {
     pub test_case_summaries: Vec<TestCaseSummary>,
     /// Status of the runner after executing tests in the file
     pub runner_exit_status: RunnerStatus,
-    /// Type of the test crate
-    pub test_crate_type: CrateLocation,
+    // /// Type of the test crate
+    // pub test_crate_type: CrateLocation,
```

Comment:
Either delete or uncomment this

Reply:
This should stay uncommented for sure

Reply:
We no longer pass CrateLocation to forge_runner, because it's irrelevant for running tests, only for printing statistics, which is handled in a different way. Removing this.